### ISSUE TITLE
Support "slack blocks"

### ISSUE THREAD

**Issue by turt2live** [turt2live/matrix-appservice-webhooks#55]
whatever that happens to be: https://api.slack.com/messaging/attachments-to-blocks

**Comment by LaysDragon** [comment]
did matrix support block actions like this?

**Comment by turt2live** [comment]
I don't believe there's a direct translation, but we can probably fudge it a bit

**Comment by LaysDragon** [comment]
I quite hope that matrix can have this function to integrating with external service XD
For now the only way I can figure out is implement command to trigger function
